hy/compiler.py

```diff
@@ -2071,14 +2071,35 @@ def rewire_init(expr):
 
         bases_expr = []
         bases = Result()
+        keywords = []
         if expressions:
             base_list = expressions.pop(0)
             if not isinstance(base_list, HyList):
                 raise HyTypeError(base_list, "Base classes must be a list.")
-            bases_expr, bases, _ = self._compile_collect(base_list)
+            bases_expr, bases, keywords = self._compile_collect(base_list, with_kwargs=True)
 
         body = Result()
 
+        if not PY3 and keywords:
+            # Dirty shim for Python 2 to make :metaclass keyword work.
```

Comment:
I'm not sure if this is worth the added complexity to the compiler. We can already use metaclasses in Python2 using `(setv __metaclass__ foo)`. Our Python2 support is ending in 2020 at the latest. It's fine to fix bugs, but we shouldn't be adding new features for it.

Reply:
I'd be fine with this if everyone else is.

Reply:
I can confirm that I would be fine with that.